citizen_science/citizen_science.md

```diff
@@ -0,0 +1,264 @@
+<!--
+
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  1.0.0
+module_template_version: 3.1.0
+language: en
+narrator: UK English Female
+title: Citizen Science
+comment:  This is an overview of citizen science for biomedical researchers.
+long_description: This module describes what citizen science is, how it is used in research, and what motivates participants to contribute. It also highlights several examples of real citizen science projects being used in biomedical research and related fields. No prior knowledge is assumed.
+estimated_time: 30 minutes
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- list several ways members of the public can contribute to scientific projects
+- identify several different factors that may motivate people to participate in citizen science
+- analyze existing citizen science projects for things like policies on collaboration, goals of the research, and strategies for implementation
```

Comment:
This learning objective is definitely focused on in the quiz, but only a little discussed in the module itself. Could some of the content in the quiz be moved to the main portion? Maybe analyze one of the articles "together" and then the other one can be a quiz question.